packages/core/src/data-editor/data-editor.tsx

```diff
@@ -3440,7 +3440,18 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
                 return gridRef.current?.damage(damageList);
             },
             getBounds: (col, row) => {
-                return gridRef.current?.getBounds(col + rowMarkerOffset, row);
+                if (col === undefined && row === undefined && scrollRef.current && canvasRef.current) {
+                    // Return the bounds of the entire scroll area:
+                    const rect = canvasRef.current.getBoundingClientRect()
+                    const scale = rect.width / scrollRef.current.clientWidth
+                    return {   
+                         x: rect.x - scrollRef.current.scrollLeft * scale,
+                         y: rect.y - scrollRef.current.scrollTop * scale,
+                         width: scrollRef.current.scrollWidth * scale,
+                         height: scrollRef.current.scrollHeight * scale,
+                     };
+                }
+                return gridRef.current?.getBounds(col !== undefined ? col + rowMarkerOffset : undefined, row);
```

Comment:
It would probably be clearer just to keep the 0 here rather than push down the default to the dataGridRef

Reply:
ok, changed it to use `col ?? 0 + rowMarkerOffset` here 👍